cmd/agent/app/reloadcheck.go

```diff
@@ -20,7 +20,6 @@ import (
 func init() {
 	// TODO: re-enable when the API endpoint is implemented
 	// AgentCmd.AddCommand(reloadCheckCommand)
-	checkCmd.SetArgs([]string{"checkName"})
```

Comment:
should it rather be `reloadCheckCommand.SetArgs([]string{"checkName"})` ?

Reply:
I think we can clean this up, yup.